src/core/meshcop/meshcop_tlvs.hpp

```diff
@@ -361,18 +362,21 @@ class ExtendedPanIdTlv : public Tlv
      * @returns The Extended PAN ID value.
      *
      */
-    const otExtendedPanId &GetExtendedPanId(void) const { return mExtendedPanId; }
+    const Mac::ExtendedPanId &GetExtendedPanId(void) const { return mExtendedPanId; }
 
     /**
      * This method sets the Extended PAN ID value.
      *
-     * @param[in]  aExtendedPanId  A pointer to the Extended PAN ID value.
+     * @param[in]  aExtendedPanId  A Extended PAN ID value.
      *
      */
-    void SetExtendedPanId(const otExtendedPanId &aExtendedPanId) { mExtendedPanId = aExtendedPanId; }
+    void SetExtendedPanId(const otExtendedPanId &aExtendedPanId)
```

Comment:
nit: should we use `ExtendedPanId` here instead of `otExtendedPanId` to be consistent with `GetExtendedPanId()`? I realize this will likely require adding `static_cast` at some call sites.